### PR TITLE
Add static top-level landing page

### LINKED ISSUE #251: Static version of the landing page
Based on current UI work at #214

### PR DESCRIPTION
**Pull request by comzeradd**
* Fixes #251

At the moment this includes some static content and assets that would gradually been removed as we extend the template models.

This includes also the rendered css from sass. This can be removed once #234 is merged and we add sass as part of the static compression pipeline.

### CODE REVIEW

File: coss/base/static/css/coss.scss [hunk 1]
```diff
@@ -0,0 +1,124 @@
+@import 'colors';
```
Comment (akatsoulas) [inline coss/base/static/css/coss.scss]: Shouldn't this file along with others that are referring to the content be part of the `home` app and not the `base`?

**Comment by akatsoulas** [comment]
Also let's merge first PR #234 and use the compressor for the assets.

**Comment by comzeradd** [comment]
PR Updated and rebased.

I split up some assets and css code to live under the `home` app. Most of the css should part of the base app, since it's an implementation of the overall coss ui theme, and parts of that will be overwritten per app (eg. colors).

**Comment by akatsoulas** [comment]
I am not sure as to why this has to go into the base app. Even if this is part of the `groups and gatherings` it should live under the home app. Base app should have only the bare minimum (eg login capabilities and nothing related to the UI)

**Comment by comzeradd** [comment]
I don't fully agree, since most of these things are not app specific (not even home app specific). But let's not waste more time on this. I moved all assets under the home app. We can revisit later if needed, when the rest of the project starts to shape.

**Pull request merged**